components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -811,19 +811,26 @@ def _get_prepared_image(request, iid, server_id=None, conn=None,
     img = conn.getObject("Image", iid)
     if img is None:
         return
-    reverses = _get_maps_enabled(r, 'reverse', img.getSizeC())
+    invert_flags = None
+    if 'maps' in r:
+        reverses = _get_maps_enabled(r, 'reverse', img.getSizeC())
+        # 'reverse' is now deprecated (5.4.0). Also check for 'invert'
+        invert_flags = _get_maps_enabled(r, 'inverted', img.getSizeC())
+        # invert is True if 'invert' OR 'reverse' is enabled
+        invert_flags = [z[0] if z[0] is not None else z[1] for z in
+                        zip(invert_flags, reverses)]
```

Comment:
File "/opt/hudson/workspace/WEB-DEV-merge-deploy/OMERO.py/lib/python/omeroweb/webgateway/views.py", line 821, in _get_prepared_image
   zip(invert_flags, reverses)]
TypeError: zip argument #1 must support iteration